pkg/server/server.go

```diff
@@ -307,3 +319,26 @@ func setNoProxyEnv(config *config.Control) error {
 	}, ",")
 	return os.Setenv("NO_PROXY", envList)
 }
+
+func writeConfigSymlink(kubeconfig, kubeconfigSymlink string) error {
+	if _, err := os.Stat(kubeconfigSymlink); err == nil {
+		if err := os.Remove(kubeconfigSymlink); err != nil {
+			return fmt.Errorf("failed to remove %s file: %v", kubeconfigSymlink, err)
+		}
+	}
+	if err := os.MkdirAll(filepath.Dir(kubeconfigSymlink), 0755); err != nil {
+		return fmt.Errorf("failed to create path for symlink: %v", err)
+	}
+	if err := os.Symlink(kubeconfig, kubeconfigSymlink); err != nil {
+		return fmt.Errorf("failed to create symlink: %v", err)
+	}
+	return nil
+}
+
+func isSymlink(config string) bool {
+	eval, err := filepath.EvalSymlinks(config)
```

Comment:
To check if something is a symlink do Lstat and then `stat.Mode() & os.ModeSymlink == os.ModeSymlink`